types/chrome/index.d.ts

```diff
@@ -4831,12 +4836,14 @@ declare namespace chrome.identity {
     /**
      * Removes an OAuth2 access token from the Identity API's token cache.
      * If an access token is discovered to be invalid, it should be passed to removeCachedAuthToken to remove it from the cache. The app may then retrieve a fresh token with getAuthToken.
+     * If `callback` is not provided, the function returns a Promise when the state has been removed from the cache.
      * @param details Token information.
      * @param callback Called when the token has been removed from the cache.
      * If you specify the callback parameter, it should be a function that looks like this:
      * function() {...};
      */
-    export function removeCachedAuthToken(details: TokenInformation, callback?: () => void): void;
+    export function removeCachedAuthToken(details: TokenInformation): Promise<void>;
+    export function removeCachedAuthToken(details: TokenInformation, callback: () => void): void;
```

Comment:
Also in the future: there should not be a `callback?` parameter, but just `callback` and a signature that returns a `Promise`. 

Many "promisified" APIs in this file incorrectly assert that if `callback` is not provided, the function still returns `void`